extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -368,4 +292,14 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		return title;
 	}
 
+	//Confirmation message dialog
+	private async confirmReplace(): Promise<boolean> {
+		return await this.prompter.promptSingle<boolean>(<IQuestion>{
+			type: QuestionTypes.confirm,
+			message: localize('confirmReplace', "Folder already exists. Are you sure you want to replace?"),
+			default: false
```

Comment:
Include the path to the folder. I'd also make sure it's clear that you'll be deleting the entire folder and then recreating it

Reply:
@yualan Could you chip in and tell me the exact text to be shown here?

Reply:
I checked offline with Alan and he suggested "Folder already exists. Are you sure you want to delete and replace this folder?"

Reply:
We should definitely be including the name of the folder unless the experience makes it really obvious. But this isn't a folder the users chooses right? So I don't think they'd have any idea what folder we're talking about.

Something like this would be better I think

`The folder '{0}' exists. Are you sure you want to delete and replace it?`

As a side note - I would also suggest that instead of just checking for it existing we also check if it's empty. That's more of a "nice to have feature" but is pretty common practice since we only really care about prompting if there's actually anything in the folder the user might care about.

Reply:
![image](https://user-images.githubusercontent.com/12754347/65343832-da8aa580-db8a-11e9-9e53-c1e9442522e3.png)

The prompt isn't showing the message in two lines, is there a way to wrap text?

Reply:
I would suggest not using the quick pick for this - just use a normal message box. This is what VS Code does for file/folder deletion currently 

![image](https://user-images.githubusercontent.com/28519865/65345967-63f0a680-db90-11e9-84e9-41ea020ec693.png)

Reply:
I'll make these changes in a separate PR. Thanks for taking the time in providing me the feedback to get these done right.